kyuubi-common/src/main/scala/org/apache/kyuubi/service/FrontendService.scala

```diff
@@ -99,7 +99,8 @@ class FrontendService private (name: String, be: BackendService, oomHook: Runnab
         s" [$minThreads, $maxThreads] worker threads")
     } catch {
       case e: Throwable =>
-        throw new KyuubiException("Failed to initialize frontend service", e)
+        throw new KyuubiException("Failed to initialize frontend service, " +
+          s"please check this host:port $serverAddr:$portNum is available.", e)
```

Comment:
this looks like, we into a particular exception inevitable

Reply:
How about just `Failed to initialize frontend service on host:port?

Reply:
Updated.